2.4-micro/root/usr/share/container-scripts/httpd/README.md

```diff
@@ -0,0 +1,248 @@
+Apache HTTP Server 2.4 Micro Container Image
+======================================
+
+This container image includes Apache HTTP Server 2.4 for OpenShift and general usage.
+Users can choose between RHEL, CentOS and Fedora based images.
+The RHEL images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/),
+the CentOS images are available on [Quay.io](https://quay.io/organization/centos7),
+and the Fedora images are available in [Fedora Registry](https://registry.fedoraproject.org/).
+The resulting image can be run using [podman](https://github.com/containers/libpod).
+
+Note: while the examples in this README are calling `podman`, you can replace any such calls by `docker` with the same arguments
+
+Description
+-----------
+
+Apache HTTP Server 2.4 available as a micro-sized container, is a powerful, efficient,
```

Comment:
We should maybe explain what micro means -- specifically that we have no yum or rpm.

Reply:
What does it mean `micro`? It is not mentioned here. What is the difference between `2.4` and `2.4-micro`. This could be misleading.